spec/web_helpers.rb

```diff
@@ -0,0 +1,7 @@
+def user_sign_up
```

Comment:
good use of a method to do user sign up. we should have created one for property listing too.